t/pt-heartbeat/multi_update_mode.t

```diff
@@ -60,175 +61,231 @@ foreach my $port (@ports) {
    );
 }
 sleep 5;
-# Check heartbeat on master.
-my $rows = $master_dbh->selectall_hashref("select * from test.heartbeat", 'server_id');
+# Check heartbeat on source.
+my $rows = $source_dbh->selectall_hashref("select * from test.heartbeat", 'server_id');
 
 is(
    scalar keys %$rows,
    1,
-   "One heartbeat row on master"
+   "One heartbeat row on source"
 );
 
 ok(
    exists $rows->{12345},
-   "Master heartbeat"
+   "Source heartbeat"
 );
 
 ok(
    defined $rows->{12345}->{file} && defined $rows->{12345}->{position},
-   "Master file and position"
+   "Source file and position"
 );
 
 ok(
-   !$rows->{12345}->{relay_master_log_file} && !$rows->{12345}->{exec_master_log_pos},
-   "No relay_master_log_file or exec_master_log_pos for master"
+   !$rows->{12345}->{"relay_${source_name}_log_file"} && !$rows->{12345}->{"exec_${source_name}_log_pos"},
+   "No relay_source_log_file or exec_source_log_pos for source"
 );
 
-# Check heartbeat on slave1.
-$rows = $slave1_dbh->selectall_hashref("select * from test.heartbeat", 'server_id');
+# Check heartbeat on replica1.
+$rows = $replica1_dbh->selectall_hashref("select * from test.heartbeat", 'server_id');
 
 is(
    scalar keys %$rows,
    2,
-   "Two heartbeat rows on slave1"
+   "Two heartbeat rows on replica1"
 );
 
 ok(
    exists $rows->{12345},
-   "Slave1 has master heartbeat",
+   "Replica1 has source heartbeat",
 );
 
 ok(
    exists $rows->{12346},
-   "Slave1 heartbeat"
+   "Replica1 heartbeat"
 );
 
 ok(
    defined $rows->{12346}->{file} && defined $rows->{12346}->{position},
-   "Slave1 master file and position"
+   "Replica1 source file and position"
 );
 
 ok(
-   $rows->{12346}->{relay_master_log_file} && $rows->{12346}->{exec_master_log_pos},
-   "Slave1 relay_master_log_file and exec_master_log_pos for master"
-);
+   $rows->{12346}->{"relay_source_log_file"} && $rows->{12346}->{"exec_source_log_pos"},
```

Comment:
info: why is this test (and next one) not using `relay_$(source_name}_log_file` like the previous one ? Can it miss an edge case ?

Reply:
Because it selects from the test.heartbeat table, and not from SHOW SLAVE/REPLICA STATUS. This table has static column names, not depending on server version. Legacy column names are supported, this is tested by different test: see t/pt-heartbeat/heartbeat_table.t

Reply:
Got it thank you